src/inject/dynamic-theme/stylesheet-modifier.ts

```diff
@@ -24,13 +24,19 @@ function getThemeKey(theme: Theme) {
 
 const asyncQueue = createAsyncTasksQueue();
 
+// 'not-found' It means dark reader didn't found any non loaded link(s) on this stylesheet.
+// 'found' It means that dark reader did found non loaded link(s) and wasn't rebuilt yet.
```

Comment:
**D**ark **R**eader **found** or **has found** (to the best of my English knowledge).